src/replica/duplication/duplication_sync_timer.h

```diff
@@ -73,6 +73,10 @@ class duplication_sync_timer
 
     std::vector<replica_ptr> get_all_replicas();
 
+    // duplication_sync_timer is async with replica close,so replica_duplicator_manager may already
+    // release dup should stop right now
```

Comment:
Just describe what does this function do is OK, you can describe why invoke it when in the caller.